### ISSUE TITLE
Unreal Engine Version?

### ISSUE THREAD

**Issue by Ademord** [NVIDIA/Dataset_Synthesizer#83]
Hello
I was wondering if there is a way to make NDDS compatible with 4.25? 
Edit: I have successfully upgraded the plugin to 4.24 but upgrading to 4.25 is giving me a headache :/ if someone understands the changes between Uproperties and Fproperties it'd be a great first step :)

**Comment by wightwhale** [comment]
I'm extremely interested in updating this to a newer unreal version. Can you share what you needed to do to upgrade the plugin to 4.24? Perhaps we could make a branch that works with newer unreal versions?

**Comment by wightwhale** [comment]
I actually see the pull request on the other tab, do I just need to just open the project in 4.24 and make the changes to the files listed in the pull request?

**Comment by wightwhale** [comment]
I was able to get it all pulled into my version yesterday. Thanks for updating it to 4.24!

**Comment by Ademord** [comment]
Yes I did the same! I couldn't finishing updating it to 4.25 unfortunately. I got blocked on some C++ errors I didn't know how to move past, like the UProperties for example.

**Comment by RiplleYang** [comment]
> I actually see the pull request on the other tab, do I just need to just open the project in 4.24 and make the changes to the files listed in the pull request?

Hello, Can you share what you needed to do to upgrade the plugin to 4.24?  It's hard to get your contact info

**Comment by HaiyiMei** [comment]
Hey guys! Any new information about updating to 4.25?

**Comment by SanggunLee** [comment]
Im porting it to unreal 4.26. I will leave a message here when completed

**Comment by Ademord** [comment]
@HaiyiMei [here](https://github.com/NVIDIA/Dataset_Synthesizer/pull/89#issuecomment-717390364) you have the pull request for 4.25

@SanggunLee you're the real MVP, thank you!

**Comment by SanggunLee** [comment]
There is a compile error in 
https://github.com/NVIDIA/Dataset_Synthesizer/pull/89
I tested with UE4.25.4

**Comment by SanggunLee** [comment]
I tested NDDS on UE 4.25.4 and it worked well.
I will upload codes.

**Comment by Ademord** [comment]
Where did you upload it @SanggunLee ? can you make a pull request as well if you need?

**Comment by SanggunLee** [comment]
> Where did you upload it @SanggunLee ? can you make a pull request as well if you need?

Check https://github.com/SanggunLee/Dataset_Synthesizer/tree/UE4.25.4
download code and just follow the NDDS.pdf

**Comment by SanggunLee** [comment]
NDDS with UE4.26 updated.
Check https://github.com/SanggunLee/Dataset_Synthesizer/tree/UE4.26
Only Window supported, not Linux.
